src/pages/loading-waiting-states/progress-bar/index.tsx

```diff
@@ -255,44 +252,38 @@ export const ProgressBar = (): JSX.Element => {
+                                    classes={{
+                                        root: classes.snackbarRoot,
+                                        anchorOriginBottomCenter: classes.bottomCenter,
+                                    }}
+                                    open={item.open}
                                     autoHideDuration={item.progress === 100 ? 3000 : null}
                                     onClose={(e, reason): void => handleRequestClose(e, reason, item.id)}
+                                    TransitionProps={{ timeout: 300, onExited: (): void => handleExited(item.id) }}
                                 >
-                                    <SnackbarContent
-                                        classes={{ root: classes.SnackbarContent }}
-                                        action={
-                                            <>
-                                                <Button
-                                                    variant="outlined"
-                                                    className={classes.closeButtonContainer}
-                                                    onClick={(): void => removeListItem(item.id, item.status)}
-                                                >
-                                                    {item.progress === 100 ? 'View' : 'Cancel'}
-                                                </Button>
-                                            </>
-                                        }
-                                        message={
-                                            <div className={classes.messageContainer}>
-                                                <div className={classes.iconContainer}>
-                                                    <Description className={classes.icon} />
-                                                </div>
-                                                <div className={classes.messageTextContainer}>
-                                                    <Typography variant={'subtitle1'} color={'inherit'}>
-                                                        {item.name}
-                                                    </Typography>
-                                                    <Typography variant={'subtitle2'} className={classes.subTitle}>
-                                                        {item.status}
-                                                    </Typography>
-                                                </div>
-                                            </div>
-                                        }
-                                    />
+                                    <div>
+                                        <MuiThemeProvider theme={createTheme(PXBThemes.blueDark)}>
+                                            <InfoListItem
+                                                key={`infolist${item.id}`}
```

Comment:
you don't need keys on these if they're not being created from an array (just the top level item (div) needs it)

Reply:
and `key` prop only needs to be different within a list, so next time you implement something like this, you would only need `key={item.id}`, and there would not be any naming confliction.